### PR TITLE
docs + config cleanup

### PR DESCRIPTION
**Pull request by themattinthehatt**
* [fiftyone]: remove "fast" dataset type, always use "slow" (i.e. include confidence)
* [fiftyone]: remove unused video plotting functionality
* [pca]: clean up config options
* [docs]: add info about data config and nan uniform heatmaps

### CODE REVIEW

**Review by danbider — COMMENTED** [review]
LGTM

File: docs/source/user_guide_advanced/unsupervised_losses.rst [hunk 1]
```diff
@@ -186,7 +186,7 @@ If instead you want to include the ears and tailbase:
       columns_for_singleview_pca: [1, 2, 4]
```
[the remark this answers is not in the document]
Comment (themattinthehatt) [reply docs/source/user_guide_advanced/unsupervised_losses.rst:186]: yes definitely agree, i might do this after we merge all of farzad/selmaan's code in. "mirrored_column_matches" could also be more descriptive

File: docs/source/user_guide_advanced/unsupervised_losses.rst [hunk 2]
```diff
@@ -203,9 +203,7 @@ Besides the ``log_weight`` none of the provided values need to be tested for new
 
 * ``log_weight``: weight of the loss in the final cost function
 * ``components_to_keep``: predictions should lie within the low-d subspace spanned by components that describe this fraction of variance
-* ``empirical_epsilon_percentile``: the reprojecton error on labeled training data is computed to arrive at a noise ceiling; reprojection errors from the video data are not penalized if they fall below this percentile of labeled data error (replaces ``epsilon``)
-* ``empirical_epsilon_multiplier``: this allows the user to increase the epsilon relative the the empirical epsilon error; with the multiplier the effective epsilon is `eff_epsilon = percentile(error, empirical_epsilon_percentile) * empirical_epsilon_multiplier`
-* ``epsilon``: absolute error (in pixels) below which pca loss is zeroed out; if not null, this parameter takes precedence over ``empirical_epsilon_percentile``
+* ``epsilon``: absolute error (in pixels) below which pca loss is zeroed out; if not null, this parameter is automatically computed from the labeled data
```
[the remark this answers is not in the document]
Comment (themattinthehatt) [reply docs/source/user_guide_advanced/unsupervised_losses.rst:206]: these are saying the same things though right?

**Comment by themattinthehatt** [comment]
closes #129

**Pull request merged**